src/Workleap.DomainEventPropagation.Abstractions/ISubscriptionDomainEventBehavior.cs

```diff
@@ -0,0 +1,8 @@
+namespace Workleap.DomainEventPropagation;
+
+public delegate Task SubscriptionDomainEventHandlerDelegate(IDomainEventWrapper domainEventWrapper, IDomainEventSubscriptionContext subscriptionContext, CancellationToken cancellationToken);
+
+public interface ISubscriptionDomainEventBehavior
```

Comment:
Should it be an abstract class? (I generally don't like interfaces as they are harder to update without breaking changes)